guide/text/15-images.md

```diff
@@ -25,7 +29,22 @@ Let's break that down:
 * `{{ site.image-set }}/` means 'go into the folder containing our preferred set of images' (as defined in `_config.yml`). The default image-set folder is `images`.
 * finally, the image file name.
 
-Many images are in figures. That is, they include an image followed by a caption. We put these together in a `<blockquote>` element with a `.figure` class. We can then control placement by styling the `<blockquote>`.
+### Images in translations
+
+If you are creating a [translation](70-translations.html#translations) in a subdirectory of `text`, and your images are in the parent book folder, you need to change the path to the images slightly. You have two options:
```

Comment:
Since we recommend `{{ path-to-book-directory }}`, should we put that first (and move the note into the list item)?